integration/helpers/instance.go

```diff
@@ -597,11 +604,26 @@ func (i *TeleInstance) CreateEx(t *testing.T, trustedSecrets []*InstanceSecrets,
 	return i.CreateWithConf(t, tconf)
 }
 
+func (i *TeleInstance) createTeleportProcess(tconf *servicecfg.Config) (*service.TeleportProcess, error) {
+	if i.ProcessProvider == nil {
+		p, err := service.NewTeleport(tconf)
+		if err != nil {
+			return nil, err
```

Comment:
nit: missing trace.Wrap here (or take the suggestion below)

```suggestion
			return nil, trace.Wrap(err)
```